src/main/java/lotto/domain/IssuedLottos.java

```diff
@@ -0,0 +1,35 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import lotto.vo.response.LottoResponse;
+
+public class IssuedLottos {
+    private final List<Lotto> lottos;
+
+    public IssuedLottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public int numberOfIssuedLottos() {
+        return lottos.size();
+    }
+
+    public List<LottoResponse> convertToResponse() {
+        return lottos.stream()
+            .map(Lotto::convertToResponse)
+            .collect(Collectors.toList());
+    }
+
+    public WinningRanks determineRanks(Lotto winnnigLotto, BonusNumber bonusNumber) {
+        List<Rank> ranks = new ArrayList<>();
+        for(Lotto lotto: lottos) {
+            int criterion = lotto.countMatching(winnnigLotto);
+            boolean hasBonusNum = bonusNumber.contained(lotto);
+            Rank rank = Rank.determineByMatchingCount(criterion, hasBonusNum);
+            ranks.add(rank);
+        }
+        return new WinningRanks(ranks);
+    }
```

Comment:
(이렇게 바꿀 수 있어요!)
```suggestion
    public WinningRanks determineRanks(Lotto winnnigLotto, BonusNumber bonusNumber) {
         return new WinningRanks(lottos.stream()
                .map(lotto -> Rank.determineByMatchingCount(
                        lotto.countMatching(winnnigLotto),
                        bonusNumber.contained(lotto)))
                .toList());
    }
```
이렇게 스트림API를 활용하는 방법도 있어요 :)